deploy/helm/omni/values.yaml

```
@@ -0,0 +1,68 @@
+
+nameOverride: omni  # optional, if ommitted the Chart name will be used
+debug: false
+domainName: omni.makomedia.nl
```

I would almost prefer `{{ required "You must define a domainName" .Values.domainName }}` where it is consumed, but it's not a requirement.
At the very least the default values should be something more like `omni.example.com`.

```suggestion
domainName: 
```

---

fixed